homeassistant/components/bmw_connected_drive/sensor.py

```diff
@@ -52,6 +54,12 @@ class BMWSensorEntityDescription(SensorEntityDescription):
         unit_metric=TIME_HOURS,
         unit_imperial=TIME_HOURS,
     ),
+    "charging_end_time": BMWSensorEntityDescription(
```

Comment:
Please separate fixes from feature additions. Add this sensor in a separate PR.

Reply:
Created https://github.com/home-assistant/core/pull/60942 & removed from this PR.